app/api/actions/github/route.ts

```diff
@@ -469,7 +469,7 @@ export async function POST(request: Request) {
       );
       // Find our bot's comment
       let botComment = comments.data.find((comment) => {
-        return comment?.user?.login.includes("watermelon-context");
+        return comment?.user?.login.includes("watermelon-copilot-for-code-review");
```

Comment:
Did we change the name of the bot?
When?

Reply:
When we agreed to change the messaging of the product and the company as a whole